server/schema/clubs.schema.ts

```diff
@@ -5,7 +5,7 @@ import {
   integer,
   pgTable,
   smallint,
-  smallserial,
+  serial,
```

Comment:
Why?

Reply:
because smallserial is not recognized by typescript as a default auto-incrementing type, and thus, it always asks to include id also in the data which should not be the case.

Reply:
This has been fixed in [`drizzle-orm v0.30.5`](https://github.com/drizzle-team/drizzle-orm/releases/tag/0.30.5). Drizzle was updated in master to the latest version, just rebase this branch and revert back to using `smallserial`

Reply:
oh, alright